### PR TITLE
[ACA-2568] Permission-List - Removal of List ARIA

### PR DESCRIPTION
**Pull request by Steady5063**
**Please check if the PR fulfills these requirements**

> - [ X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ X] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Currently items in the table are reading as a list.

**What is the new behaviour?**
The items in the table will maintain their table markup and semantic meaning.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+@Directive({
+    selector: '[adf-list-a11y]'
+})
+export class MatChipDirective {
+    constructor(private el: ElementRef, private renderer: Renderer2) {}
+
+    ngAfterViewInit() {
+        setTimeout(() => {
+            this.renderer.removeAttribute(this.el.nativeElement, 'role');
```
Comment (pionnegru) [inline lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts]: what would be the difference with what we do [here](https://github.com/Alfresco/alfresco-ng2-components/pull/5416/files#diff-a19aedfa9dcd553f1e38dedc522a1f02R29-R31)

File: lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            this.renderer.removeAttribute(this.el.nativeElement.querySelector('.mat-chip'), 'role');
```
Comment (pionnegru) [inline lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts]: [attr.role]="null" on <mat-chip> could do this trick

File: lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts (hunk 1 above)
Comment (Steady5063) [reply lib/content-services/src/lib/permission-manager/components/mat-chip.directive.ts]: If we end up making Chris' change global, then we would not need this specifically. We would just need to add the attribute he uses for selector on this and it should work

**Comment by DenysVuika** [comment]
Closing this PR as Google already provides all the necessary aria elements for this component:
https://material.angular.io/components/chips/overview#accessibility

```
A chip-list behaves as a role="listbox", with each chip being a role="option". 
The chip input should have a placeholder or be given a meaningful label via aria-label or aria-labelledby.
```

**Pull request closed without merging**